.github/workflows/CI.yml

```diff
@@ -60,8 +62,14 @@ jobs:
         }
         Write-Host "Runner started"
         $url = "$runner_host/run?notimers=true&$env:JQUERY&constellation=$env:CONSTEL"
-        & "C:\Program Files\Internet Explorer\iexplore.exe" "$url"
-        Write-Host "IE started"
+        $browser = $env:BROWSER
+        $browserPaths = @{
+          IE = "C:\Program Files\Internet Explorer\iexplore.exe";
+          MSEdge = "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe";
+        }
+        Write-Host "Starting $browser with $url"
+        Start-Process -File $browserPaths[$browser] -ArgumentList "$url"
```

Comment:
```suggestion
        Start-Process -File "$browserPaths[$browser]" -ArgumentList "$url"
```

Reply:
Done in https://github.com/DevExpress/DevExtreme/pull/15033/commits/cbc21eca69966f45fb1529015c47fdda93490455.